config/clients/js/template/README_initializing.mustache

```diff
@@ -27,3 +27,25 @@ const {{appCamelCaseName}} = new {{appShortName}}Api({
   }
 });
 ```
+
+#### Client Credentials
```

Comment:
Can you add an issue in https://github.com/openfga/openfga.dev/issues to ensure we add this to our docs once we document in GoLang, Python and .NET?

Reply:
Feel free to resolve this comment

Reply:
https://github.com/openfga/openfga.dev/issues/383